modules/node_modules/@ncigdc/uikit/Dropdown.js

```diff
@@ -16,22 +16,28 @@ const Dropdown = ({
   mouseUpHandler,
   children,
   dropdownStyle = {},
+  dropdownClassName = '',
   itemStyle = {},
+  dropdownItemClass = true,
+  buttonWithIcon = null,
```

Comment:
I think `button` is probably better term here. you're passing in an icon from the outside, but this component doesn't care if it's an icon or not